lib/vacations.ex

```diff
@@ -0,0 +1,143 @@
+defmodule GlassFactoryApi.Vacations do
+  @moduledoc """
+  Provides the methods to get vacations of all or a specific user.
+  """
+
+  alias GlassFactoryApi.ApiClient
+  alias GlassFactoryApi.Vacations.Vacation
+
+  @doc """
+  Return a tuple with `:ok` atom and an list of vacations.
+  or an tuple with `:error` and a string with the error description.
+
+  It's possible apply some filters, like user_id, start and end to limit dates, and vacation_type
```

Comment:
We can also use bullet lists to document each option individually. Take a look at [`String.split/3`](https://hexdocs.pm/elixir/String.html#split/3) for a good example.